example/linux/Makefile

```diff
@@ -88,6 +84,9 @@ LDFLAGS=-L$(BUNDLE_LIB_DIR) \
 
 # Targets
 
+.PHONY: all
+all: $(FLUTTER_ROOT)/packages/flutter_tools/bin/linux_backend.sh
```

Comment:
Also, the `$(FLUTTER_ASSETS_SOURCE)` rule below that currently calls `build build` should be removed, and `$(FLUTTER_ASSETS_SOURCE)` should become part of the left hand side of the rule depending on `flutterbuild`

Reply:
I tried to create a separate sync target and then remove the dependency on build bundle so we dont call it twice

Reply:
You still have necessary files that don't have generation rules (everything that's still depending on the now-non-existent `$(FLUTTER_COPY_STAMP_FILE)` variable in your current patch; many scenarios would still fail. For instance, try, in a completely clean setup, running `make build/linux/debug/flutter_desktop_example`. (That's an unlikely scenario, but the same thing happens very easily if someone passes `-j`.)

I thought an order-only PHONY dependency wouldn't run multiple times; could you post the version with the dependencies where you saw the backend script firing repeatedly?

(Specifically, I thought this would work:
```
$(WRAPPER_SOURCES) $(FLUTTER_LIB): | sync

...

bundleflutterassets: | sync
```